### ISSUE TITLE
Edit on error setting does not allow manual entry when url is bad

### ISSUE THREAD

**Issue by westshawn** [grundic/teamcity-web-parameters#14]
web-parameters v 1.6.2
TC server 10.0.2

```
webPopulatedSelect display='prompt' format='json' url='http://a.website/with/unreachable/page/' enableEditOnError='true'
```
Click Run or "..."
```
Unexpected status code 500
Internal Server Error
No text box for manual value input
```

Note: this manual edit on error functionality was also broken in v1.4, but in a different way:
Instead of nothing for manual value input, there was a filter control on an empty list
text= "No results found"
Typing into the list filter just attempted to filter on already empty list, but did not accept a value to pass to the build.

**Comment by grundic** [comment]
@westshawn, I'll check on this issue and would try so find a fix for it.
Thank you for your bug reports!

**Comment by grundic** [comment]
I fixed this issue in [v1.6.3](https://github.com/grundic/teamcity-web-parameters/releases/tag/v1.6.3), please have a look.
Thank you!

**Comment by westshawn** [comment]
I verified user can enter a value and the build runs with the manually entered value when web-param fails due to a bad url.
Spasibo @grundic !
